<a href="https://colab.research.google.com/github/LuisaMariaMontoyaConde1994/TrabajoClaseestadistica/blob/main/actividad_1.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [ ]:
#Daniel Felipe Castillo

In [4]:
import numpy as np
import matplotlib.pyplot as plt
from scipy.stats import binom

En una elección, un candidato tiene una probabilidad del $60\%$ de recibir un voto en cada votación individual. Si se emiten 20 votos.
1. cual es la probabilidad de que al menos obtenga 12 votos?
2. Cual es la probabilidad de que a lo mas obtenga 5 votos?
3. cual es la probabilidad de que no obtenga votos?
4. cual es la probabilidad que obtenga entre 10 y 8 votos?

respuesta1

In [7]:
n=20
p=0.6
x=12
prob= 1-binom.cdf(x-1,n,p)
m=prob*100
print(prob)
print('la probabilidad es' ,m,'%')

0.5955987253122248
la probabilidad es 59.55987253122248 %


La probabilidad de que al menos tenga 12 votos es de 59.55987253122248 %

rta2

In [8]:
n=20
p=0.6
x=5
prob=binom.cdf(x,n,p)
m=prob*100
print(prob)
print('la probabilidad es' ,m,'%')

0.001611524643456288
la probabilidad es 0.1611524643456288 %


La probabiliodad de que a lo mas obtenga 5 votos es de 0.1611524643456288 %

rta3

In [9]:
n=20
p=0.6
x=0
prob=binom.pmf(x,n,p)
m=prob*100
print(prob)
print('la probabilidad es' ,m,'%')

1.0995116277760013e-08
la probabilidad es 1.0995116277760013e-06 %


La probabilidad de que no obtenga votos es de 1.0995116277760013e-06 %

rta4

In [11]:
n=20
p=0.6
x=10
prob=binom.cdf(x,n,p)
m=prob*100
print(prob)
print('la probabilidad es' ,m,'%')

0.24466279668360685
la probabilidad es 24.466279668360684 %


In [12]:
n=20
p=0.6
x=8
prob=binom.cdf(x,n,p)
m=prob*100
print(prob)
print('la probabilidad es' ,m,'%')

0.056526367034253025
la probabilidad es 5.652636703425302 %


In [14]:
(0.24466279668360685-0.056526367034253025)*100

18.813642964935383

La probabilidad que obtenga entre 8 y 10 votos es de 18.813642964935383%